lisa/tools/dmesg.py

```diff
@@ -43,8 +43,6 @@ def check_kernel_errors(
         throw_error: bool = True,
     ) -> str:
         command_output = self._run(force_run=force_run)
-        if command_output.exit_code != 0:
```

Comment:
It's lost.

Reply:
The check is moved to _run function
https://github.com/microsoft/lisa/pull/2196/files#diff-a66e4e010a83a2d730960fa1fa9f07818ac40a963321177890c49367022e462bR89


Should it be reverted?

Reply:
If it's moved into _run, it means all calls to _run will assert the exit code. It changes the current behavior.